sys-process/bottom/files/bottom-0.9.6-bump-deps.patch

```diff
@@ -0,0 +1,101 @@
+From ea863b17485382259c5faef8e2f48c3b2f07ce49 Mon Sep 17 00:00:00 2001
+From: Randy Barlow <randy@electronsweatshop.com>
+Date: Thu, 25 Jul 2024 20:25:33 -0400
+Subject: [PATCH] Update time to 0.9.6
```

Comment:
I added this patch in with this Rust PR since it is needed for Rust 1.80.0. I'm happy to split it out into a separate pull request if that's preferred.

Reply:
(I added two more bottom patches to deal with lints. Again, happy to split those to another PR if desired...)

Reply:
I went ahead and moved bottom to its own PR. I also tested that that PR does build with this Rust 1.80 build.

See https://github.com/gentoo/gentoo/pull/37889